README.rst

```diff
@@ -21,6 +21,8 @@ Please use the `main ZMON repository`_ to start a Vagrant demo box.
 
 Make sure the provided Vagrant-Box is up and all services are running.
 
+Update your GitHub ``client-id`` and ``client-secret`` in ``zmon-controller-app/src/main/config/application-github.yml``.
```

Comment:
this should not be recommended, use environment variables instead! See https://github.com/zalando/zmon/blob/master/vagrant/start-services.sh#L35